archinstall/lib/user_interaction.py

```diff
@@ -424,24 +424,38 @@ def read_packages(already_defined: list = []) -> list:
 	return packages
 
 
-def ask_to_configure_network() -> Dict[str, Any]:
+def ask_to_configure_network(preset :Dict[str, Any] = None) -> Dict[str, Any]:
 	# Optionally configure one network interface.
 	# while 1:
 	# {MAC: Ifname}
-
-	iso_config = _('Copy ISO network configuration to installation')
-	network_manager = _('Use NetworkManager (necessary to configure internet graphically in GNOME and KDE)')
+	iso_config = str(_('Copy ISO network configuration to installation'))
+	iso_config_nt = 'Copy ISO network configuration to installation' # do not translate
```

Comment:
This could be rewritten to be a little bit less repetitive.

iso_config = str(_(iso_config_nt))
network_manager = str(_(network_manager_nt))

Reply:
I've adopted @svartkanin 's version, so this is solved